src/Cygnus/DrushExport/AbstractExport.php

```diff
@@ -757,4 +772,18 @@ protected function importContentNodes($limit = 100)
             $count -= $limit;
         }
     }
+    
+    // storage of legacy values pre-drush
+    public function convertLegacy(&$node) 
```

Comment:
Functionally doesn't matter, but in PHP5 all object pointers are passed by ref by default. So `&$node`  in the function definition is redundant, assuming the value of `$node` is an object. This should likely also have a typehint to indicate that case, such as `convertLegacy(\StdClass $node)`. Not a necessary change here, but worth keeping in mind for future code